LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
should we ask Max which license he recommends?

Reply:
merging but will ask next week